src/redux/users-reducer.js

```diff
@@ -108,4 +110,42 @@ export const toggleFollowingProgress = (isFetching, userID) => ({
   userID
 });
 
+export const getUsers = (currentPage, pageSize) => {
+  return (dispatch) => {
+    dispatch(toggleIsFetching(true));
+    dispatch(setCurrentPage(currentPage));
+    userAPI.getUsers(currentPage, pageSize).then(data => {
+      dispatch(toggleIsFetching(false));
+      dispatch(setUsers(data.items));
+      dispatch(setTotalUsersCount(data.totalCount));
+    });
+  }
+}
```

Comment:
As you can see, you've moved piece of logic in separate function. Now we can use the function and forgot how to get users.